<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_desktop</class>
 <widget class="QWidget" name="main_desktop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>main_desktop</string>
  </property>
  <widget class="QPushButton" name="cameraButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="lockButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="recordButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="setFirstButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>169</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="movieButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>170</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="camera_change_Button">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>170</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="time_Label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(42, 56, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="unknowButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>unknow</string>
   </property>
  </widget>
  <widget class="QPushButton" name="compassButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>正北</string>
   </property>
  </widget>
  <widget class="QLabel" name="cameraView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>618</width>
     <height>215</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>cameraView</zorder>
  <zorder>cameraButton</zorder>
  <zorder>lockButton</zorder>
  <zorder>recordButton</zorder>
  <zorder>setFirstButton</zorder>
  <zorder>movieButton</zorder>
  <zorder>camera_change_Button</zorder>
  <zorder>time_Label</zorder>
  <zorder>unknowButton</zorder>
  <zorder>compassButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
